src/client/state.ml

```diff
@@ -231,6 +230,12 @@ let mk_unary_handler (self : t) ?buf_pool ~timer ~oc ~encoding ~timeout_s rpc :
 
 let default_timeout_s_ : float = 30.
```

Comment:
Controversial, I know, but I would put the `30.` into the `.mli` so that it's visible in the declaration. I think the only reason not to so is when it's a global constant that can be adjusted.